tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -96,7 +102,7 @@ Status RetryingFileSystem::NewRandomAccessFile(
   TF_RETURN_IF_ERROR(CallWithRetries(std::bind(&FileSystem::NewRandomAccessFile,
                                                base_file_system_.get(),
                                                filename, &base_file)));
-  result->reset(new RetryingRandomAccessFile(std::move(base_file)));
+  result->reset(new RetryingRandomAccessFile(std::move(base_file), 0));
```

Comment:
I see what you mean. Should have updated only in tests. Accidentally updated here as well. My bad. Will revert these changes.